### ISSUE TITLE
getting Nullpointer exception on Fabric

### ISSUE THREAD

**Issue by xardox69** [teads/TeadsSDK-android#138]
**Sdk version: 4.2.3**
Stack trace:

```
Caused by java.lang.NullPointerException: Attempt to read from field 'android.content.pm.ActivityInfo android.content.pm.ResolveInfo.activityInfo' on a null object reference
       at tv.teads.sdk.android.engine.ui.browser.BrowserActivity.b + 267(BrowserActivity.java:267)
       at tv.teads.sdk.android.engine.ui.browser.BrowserActivity.onCreate + 108(BrowserActivity.java:108)
       at android.app.Activity.performCreate + 7441(Activity.java:7441)
       at android.app.Activity.performCreate + 7431(Activity.java:7431)
       at android.app.Instrumentation.callActivityOnCreate + 1286(Instrumentation.java:1286)
       at android.app.ActivityThread.performLaunchActivity + 3343(ActivityThread.java:3343)
       at android.app.ActivityThread.handleLaunchActivity + 3548(ActivityThread.java:3548)
       at android.app.servertransaction.LaunchActivityItem.execute + 86(LaunchActivityItem.java:86)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks + 108(TransactionExecutor.java:108)
       at android.app.servertransaction.TransactionExecutor.execute + 68(TransactionExecutor.java:68)
       at android.app.ActivityThread$H.handleMessage + 2155(ActivityThread.java:2155)
       at android.os.Handler.dispatchMessage + 109(Handler.java:109)
       at android.os.Looper.loop + 207(Looper.java:207)
       at android.app.ActivityThread.main + 7539(ActivityThread.java:7539)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run + 524(RuntimeInit.java:524)
       at com.android.internal.os.ZygoteInit.main + 958(ZygoteInit.java:958)
```

**Comment by HugoGresse** [comment]
Hi,

Thanks for the report, this will be fixed in the next release 4.3.0 coming later next month.

**Comment by HugoGresse** [comment]
Hi,
Teads SDK v4.3.0 has been released and adress this issue. I'm closing this but don't hesitate to answer here or open a new issue if needed.
